### PR TITLE
Fix the ruler alertmanager_client documentation

### PR DESCRIPTION
**Pull request by SSvilen**
**What this PR does / why we need it**:
Fixes the alertmanager_client documentation part for the ruler . 

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

**Checklist**
- [x] Reviewed the `CONTRIBUTING.md` guide
- [ ] Documentation added
- [ ] Tests updated
- [ ] `CHANGELOG.md` updated
- [ ] Changes that require user attention or interaction to upgrade are documented in `docs/sources/upgrading/_index.md`

### CODE REVIEW

File: docs/sources/configuration/_index.md [hunk 1]
```diff
@@ -667,16 +667,14 @@ alertmanager_client:
...
-  authorization:
-    # Sets the authentication type.
-    [type: <string> | default: Bearer]
-    # Sets the credentials. It is mutually exclusive with
-    # `credentials_file`.
-    [credentials: <secret>]
-    # Sets the credentials to the credentials read from the configured file.
-    # It is mutually exclusive with `credentials`.
-    [credentials_file: <filename>]
+  # Sets the authentication type.
```
Comment (MichelHollands) [inline docs/sources/configuration/_index.md:670]: The `basic_auth_username` and `basic_auth_password` mentioned just above should be on the same level as the ones you changed here I think. There is also a TLS field on NotifierConfig whose fields are not added here.

Comment (SSvilen) [reply docs/sources/configuration/_index.md:670]: The ```basic_auth_username``` and ```basic_auth_password``` are on the same level. I just removed the ```authorization``` keyword and moved the config options to the left.

Yes, the TLS options are missing. I could add those too

**Comment by jeschkies** [comment]
@SSvilen could you rebase your branch? Out soups make the build green.

**Review by jeschkies — APPROVED** [review]

**Comment by ashwanthgoli** [comment]
@SSvilen these configs are already defined here https://github.com/SSvilen/loki/blob/47c4210774fe495ba127f874804c549285e7e620/docs/sources/configuration/_index.md?plain=1#L915

what's the reason for defining these again?

**Comment by SSvilen** [comment]
tbh, I'm not sure anymore. May be it did not work? 
I've switched jobs since and can not tell for sure. 
But I can try to test it in my lab.

**Pull request merged**